### PR TITLE
Add is_approved flag to Action/Event/Vendor models

### PR DESCRIPTION
**Pull request by BradHN1**
#### Requirements (place an `x` in each `[ ]`)
* [x] I've read and understood the [Contributing Guidelines](/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've tested my changes manually.
* [ ] I've added unit tests to my PR.

##### Transparency ([Project board](https://github.com/orgs/massenergize/projects/7/views/2))
* [x] I've given my PR a meaningful title.
* [x] I linked this PR to the relevant issue.
* [x] I moved the linked issue from the "Sprint backlog" to "In progress" when I started this.
* [x] I moved the linked issue to "QA Verification" now that it is ready to merge.

### CODE REVIEW

File: src/database/models.py [hunk 1]
```diff
@@ -1110,7 +1110,6 @@ def full_json(self):
         data = self.simple_json()
         # Q: should this be in simple_json?
         data["communities"] = [c.simple_json() for c in self.communities.all()]
-        # data["admins"] = [a.simple_json() for a in self.admins.all()]
```
Comment (Opoku-Agyemang) [inline src/database/models.py:1113]: any reason we are removing this?

**Review by Opoku-Agyemang — COMMENTED** [review]
How will this also affect the queries we make to actions.list etc?  Will is_approved be used to filter?

File: src/database/models.py (hunk 1 above)
Comment (BradHN1) [reply src/database/models.py:1113]: Because it is a comment and the admins are returned by the simple_json method.

**Comment by BradHN1** [comment]
@Opoku-Agyemang 

> How will this also affect the queries we make to actions.list etc? Will is_approved be used to filter?
Good question.  Right now those queries work the same way as they do for Testimonials, which is that is_approved is ignored and only items with is_published=True are returned unless you are in sandbox. 

 Also right now the 'is_approved' is not required to be set true for is_published to be set to true; I am going to fix this so can hold off on this PR for the moment.

**Review by Davidjwkinghorn — APPROVED** [review]
LGTM

**Pull request merged**